dir.targets

```diff
@@ -70,9 +80,45 @@
     <XunitOptions Condition="'$(OS)'=='FreeBSD'">$(XunitOptions) -notrait category=nonfreebsdtests</XunitOptions>
   </PropertyGroup>
 
+    <!-- Create a deployment directory and copy the build output to it -->
+  <Target Name="CreateDeploymentDirectory">
```

Comment:
Rename to `CopyToDeploymentDirectory`?  Ditto below for `CopyToTestDeploymentDirectory`?